api-test/internal/e2e/e2e.go

```
@@ -0,0 +1,731 @@
+	q.Set("client_id", clientID)
+	q.Set("scope", strings.Join(sc.Scopes, " "))
+	q.Set("redirect_uri", redirectURI)
+	q.Set("state", state)
+	q.Set("nonce", nonce)
+	q.Set("code_challenge", challenge)
+	q.Set("code_challenge_method", "S256")
+	if len(r.acrForClaims()) > 0 {
+		q.Set("acr_values", strings.Join(r.acrForClaims(), " "))
+	}
+	if len(sc.UserinfoClaims) > 0 {
+		cj, _ := json.Marshal(map[string]any{"userinfo": sc.UserinfoClaims})
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `any` found

(errchkjson)